VersionControlUI/src/commands/CDiff.cpp

```diff
@@ -63,7 +63,7 @@ Interaction::CommandResult* CDiff::executeNamed(Visualization::Item*, Visualizat
 	QString managerName = headManager->name();
 
 	// TODO add functionality to specify two versions
-	VersionControlUI::DiffManager diffManager{name, "HEAD", managerName};
+	VersionControlUI::DiffManager diffManager{name, "HEAD", managerName, Model::SymbolMatcher{"Class"}};
```

Comment:
So this is OK for now, but in the end we should have a different way to provide the symbol matcher from the OOModel plugin. Put a TODO in the gist.